docs/ops.md

```diff
@@ -203,7 +203,8 @@ See [containerd's Plugin documentation](./PLUGINS.md)
 The linux runtime allows a few options to be set to configure the shim and the runtime that you are using.
 
 ```toml
-[plugins.linux]
+[plugins]
```

Comment:
- Please prepend `version = 2` header to clarify the syntax version
- Please clarify that "io.containerd.runtime.v1.linux" is deprecated in favor of "io.containerd.runc.v2", and will be removed in containerd 2.0.
-